STAR-Delegation/draft-ietf-acme-star-delegation.md

```diff
@@ -694,6 +695,11 @@ ACME Delegation process:
 We note that all the above messages are authenticated, and therefore each proxy
 must be able to authenticate any subordinate server.
 
+# CA Behavior
```

Comment:
I was wondering whether this should go under "Protocol Flow" rather than being a section on its own.  But I think your choice is correct.